test/widgets/actions_test.dart

```diff
@@ -182,8 +182,11 @@ void main() {
 
       final pushedRoutes = <Route<dynamic>>[];
       testNavObserver.onPushed = (route, prevRoute) => pushedRoutes.add(route);
-      final account1Route = InboxPage.buildRoute(accountId: account1.id);
-      final account2Route = InboxPage.buildRoute(accountId: account2.id);
+      // TODO(#737): switch to a realistic setup:
+      final account1Route = MaterialAccountWidgetRoute(
```

Comment:
bump https://github.com/zulip/zulip-flutter/pull/1076#discussion_r1879099666 :slightly_smiling_face:

Reply:
Ohh! I was wondering why there was only a colon added (which made some sense to me anyway).